kapitan/targets.py

```diff
@@ -444,10 +447,14 @@ def compile_target(target_obj, search_paths, compile_path, ref_controller, inven
             input_compiler = copy_compiler
         elif input_type == "remove":
             input_compiler = remove_compiler
+        elif input_type == "binary":
+            input_compiler = binary_compiler
+            if "binary_args" in comp_obj:
```

Comment:
why just not `args`?

Reply:
I was doing this to stick with the convention established in the helm input since no other input_type uses that arg but I have no problem changing it.